### ISSUE TITLE
nbsp check with false positive

### ISSUE THREAD

**Issue by jengelh** [rpm-software-management/rpmlint#553]
In Staging:M, one can observe

```
[  153s] cups.spec:1084: W: non-break-space line 1084, char 37
[  153s] The spec file contains a non-break space, which looks like a regular space in
[  153s] some editors but can lead to obscure errors. It should be replaced by a
[  153s] regular space.
```

However, the .spec file only has 677/678 lines.

**Comment by scarabeusiv** [comment]
Nah thats correct, it sees it in the converted spec that contains also the %changelog entries.
